lib/ansible/modules/windows/win_disk_facts.ps1

```diff
@@ -39,7 +36,7 @@ try {
     Fail-Json -obj $result -message "Failed to search the disks on the target: $($_.Exception.Message)"
 }
 [int32]$diskcount = $disks | Measure-Object | Select-Object  -ExpandProperty Count
-$result.ansible_facts.total_disks = $diskcount
+#$result.ansible_facts.total_disks = $diskcount
```

Comment:
May as well just remove the line if you are not going to return it anymore.

Reply:
That was the intention, thanks for catching this :-)